mono/eglib/gstr.c

```diff
@@ -205,6 +205,13 @@ Last this was checked was June-2017.
 
 Apple is at 106.
 Android is at 133.
+
+Haiku starts numbering at 0x8000_7000 (like HRESULT on Win32) for POSIX errno,
```

Comment:
HRESULTs do not start at 0x8000_7000. Please remove bogus comment.
HRESULTs span the entire 32bit range.

Reply:
Comment is unclear. It starts at `0x8000_0000` but POSIX errno begins at the +`0x7000` region.

Reply:
HRESULT starts at 8000000 just because it is 32bit signed integer. But comparing Haiku errno to Win32 HRESULT does not seem useful.